### PR TITLE
Obi 20

### PR DESCRIPTION
**Pull request by alxbenz**

### CODE REVIEW

File: src/components/releaseNote/ReleaseNote.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+	useEffect(() => {
+		if (hasSeenReleaseNote) {
+			setShowRelaseNote(false);
+		} else {
+			getMarkdown();
+		}
+	}, [hasSeenReleaseNote]);
+
+	console.log(showReleaseNote);
```
Comment (web-mi) [inline src/components/releaseNote/ReleaseNote.tsx]: REM
```suggestion
```

File: src/components/releaseNote/ReleaseNote.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+	if (!showReleaseNote || hasSeenReleaseNote) return <></>;
```
Comment (web-mi) [inline src/components/releaseNote/ReleaseNote.tsx]:
```suggestion
	if (!showReleaseNote || hasSeenReleaseNote) return null;
```

File: cypress/integration/video-calls.spec.ts [hunk 3]
```diff
@@ -19,6 +19,7 @@ describe('Video calls', () => {
 	});
 
 	beforeEach(() => {
+		cy.intercept('GET', '/releases/*', { statusCode: 404 });
```
Comment (web-mi) [inline cypress/integration/video-calls.spec.ts]: You could move this to mockApi.ts with 
`defaultReturns['releases']={ statusCode: 404}`
and
```
cy.intercept('GET', '/releases/*, (req) => {
		req.reply({
			...defaultReturns['releases'],
			...(overrides['releases'] || {})
		});
	}).as('releases');
```
And then just use cy.willReturn('releases', ...);
where you need it in your tests. Then we won't have problems in other tests where it should not return anything. ;-)

File: src/components/overlay/overlay.styles.scss [hunk 4]
```diff
@@ -198,14 +198,15 @@ $overlay-illustration-size-large: 160px;
 		@include breakpoint($fromLarge) {
 			$size: $overlay-illustration-size-from-large + $grid-base * 2;
 
-			position: absolute;
 			display: inline-block;
 			background-color: $white;
-			top: -$size / 2;
+			margin-top: ((-$size / 2) - $grid-base-six);
```
Comment (web-mi) [inline src/components/overlay/overlay.styles.scss]: Is this a fix for the overlay with illustration cut off?
Haven't seen it so i made one in OBI-13 too with fixing scroll etc. :-)

File: src/components/releaseNote/ReleaseNote.tsx [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+export const ReleaseNote: React.FC<ReleaseNoteProps> = () => {
+	const [showReleaseNote, setShowRelaseNote] = useState(false);
+	const [hasSeenReleaseNote, setHasSeenReleaseNote] = useState(false);
+	const [checkboxChecked, setCheckboxChecked] = useState(false);
+	const [releaseNoteText, setReleaseNoteText] = useState('');
+
+	const getMarkdown = async () => {
+		const response = await fetch(
+			`${uiUrl}/releases/v${packageInfo.version}.md`
```
Comment (web-mi) [inline src/components/releaseNote/ReleaseNote.tsx]: Move url to config.ts
```suggestion
			`${config.urls.releases}/v${packageInfo.version}.md`
```

**Review by web-mi — CHANGES_REQUESTED** [review]

**Review by web-mi — APPROVED** [review]

File: cypress/integration/video-calls.spec.ts (hunk 3 above)
Comment (web-mi) [inline cypress/integration/video-calls.spec.ts]: This line could be removed now?
```suggestion
```

**Review by web-mi — CHANGES_REQUESTED** [review]

File: cypress/integration/video-calls.spec.ts (hunk 3 above)
Comment (alxbenz) [reply cypress/integration/video-calls.spec.ts]: I can't find this anymore... Did you perhaps check an older commit?

**Review by web-mi — DISMISSED** [review]

**Review by web-mi — APPROVED** [review]

**Pull request merged**